source/_layouts/page.html

```diff
@@ -2,6 +2,13 @@
 layout: default
 ---
 
+{% if page.toc %}
+{% include guides.html %}
+ 
+</script>
```

Comment:
Why is this script dangling here and not part of the `guides.html` include?

Reply:
no, I think that was a copy paste error.